docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc

```diff
@@ -6,6 +6,7 @@ This section lists all updates to prebuilt detection rules, made available with
 
 To update your installed rules to the latest versions, follow the instructions in <<update-prebuilt-rules>>.
 
+For previous rule updates, please navigate to the https://www.elastic.co/guide/en/security/8.8/prebuilt-rules.html[last version].
```

Comment:
We need to have a link to the previous downloadable rule updates instead of all rules reference here.